core/src/components/select/select.tsx

```diff
@@ -204,6 +204,15 @@ export class Select implements ComponentInterface {
    */
   @Prop({ mutable: true }) value?: any | null;
 
+  /**
+   * Set to `"small"` for a select with less height,
+   * `"medium"` for the default select height,
+   * or `"large"` for a select with more height.
+   *
+   * Defaults to `"medium"` for the ionic theme, and  undefined for all other themes.
+   */
```

Comment:
```suggestion
  /**
   * The size of the select. If "large" it will increase the height of the select, while
   * "small" and "medium" provide progressively smaller heights. The default size is "medium".
   *
   * Defaults to `"medium"` for the ionic theme, and  undefined for all other themes.
   */
```

This matches the documentation in textarea more closely: https://github.com/ionic-team/ionic-framework/blob/bc3d30c3d28e617d76e6333f5593175d54060aad/core/src/components/textarea/textarea.tsx#L258-L262

Reply:
Good catch!